unstructured/partition/pdf.py

```diff
@@ -334,6 +329,9 @@ def _partition_pdf_or_image_local(
     ocr_mode: str = OCRMode.FULL_PAGE.value,
     model_name: Optional[str] = None,
     metadata_last_modified: Optional[str] = None,
+    extract_images_in_pdf: bool = False,
```

Comment:
shouldn't these be floated up to the `partition_pdf` function to make it a bit easier for end users to consume?

Reply:
(i'd go as far to say they should be floated up to partition, and the pdf-only kwargs popped for all other partition_ functions)

Reply:
Floated up in [d777dc2](https://github.com/Unstructured-IO/unstructured/pull/1810/commits/d777dc242ea51577b14b65134c0c68439315fb18) and [4c0142a](https://github.com/Unstructured-IO/unstructured/pull/1810/commits/4c0142ad4dae66b1dd4cbc8646ad1f2edf7af03f).